ts/src/bitrue.ts

```diff
@@ -248,11 +248,6 @@ export default class bitrue extends Exchange {
                 'networks': {
                     'ERC20': 'ETH',
                     'TRC20': 'TRX',
-                    'TRON': 'TRX',
-                },
-                'networksById': {
-                    'TRX': 'TRC20',
```

Comment:
@ttodua can we leave it empty with a comment like "will be automatically filled in". So the devs in the future don't try to add it again

Reply:
actually, I think that in team we should tell (and preferably announce after completion of all network-unification PRs).
then, I was thinking to automatically inject them (somewhat like we do autosort of ".has" dict). at this moment, i might not include that comment and after we merge all these PR's I can add that comment manually for all exchanges in one small PR (instead of editing 15+ PRs now)

Reply:
I'll do that same day when we finish & merge unification PRs